code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -121,6 +121,13 @@
 	if(.) //We moved
 		handle_exertion()
 		handle_leg_damage()
+	
+		if(client)
+			var/turf/T = src.loc
+			if (!isturf(T))	return
+
+			var/turf/B = GetAbove(T)
```

Comment:
Likewise, not checking `HasAbove()` and not checking if `GetAbove()` returned a usable value.